cli_reference/get_started_cli.adoc

```diff
@@ -52,25 +63,36 @@ users:
 - name: myuser
   user:
     token: NmUyM2M4YjQtZDc2ZC00MDFiLWIxZGEtMjY4N2RlZmJlODE2
----- 
+----
+====
+
+From now on, every subsequent command will make use of the server, session token
+and project referenced in that config file. Notice what happens if you run `osc
+login` again:
 
-From now on, every subsequent command will make use of the server, session token and project referenced in that config file. Notice what happens if you run `osc login` again:
+====
 
+[options="nowrap"]
 ----
 $ osc login
 Already logged into 'https://my-openshift-server.com' as 'myuser'.
 Using project 'project01'.
 ----
+====
 
-For more information check the link:../using_openshift/authentication.html#command-line-authentication[Authentication] section.
+For more information, see the link:../dev_guide/authentication.html[Authentication] section.
 
 == Configuration Files
 
-Configuration file permanently stores `osc` options; it contains a series of authentication mechanisms and server connection information associated with nicknames. As described in the previous section, the `osc login` command is capable of creating and managing configuration files automatically.
+Configuration file permanently stores `osc` options; it contains a series of
+authentication mechanisms and server connection information associated with
```

Comment:
@adellape Should this start with "Configuration files permanently store `osc` options"? Or maybe "A configuration file permanently stores `osc` options"?